specs/mrtcore/PrimaryLanguageOverride/PrimaryLanguageOverride.md

```diff
@@ -0,0 +1,121 @@
+# ApplicationLanguages.PrimaryLanguageOverride
+
+This feature provides the ability to change the application language for WindowsAppSDK applications,
+comparable to the one in
+[Windows.Globalization.ApplicationLanguages.PrimaryLanguageOverride](https://learn.microsoft.com/uwp/api/windows.globalization.applicationlanguages).
+The benefit of this addition is that it applies to both packaged and un-packaged WindowsAppSDK
+applications, whereas the existing one is not supported for un-packaged.
+
+# Background
+
+Changing the application language using
+[PrimaryLanguageOverride](https://learn.microsoft.com/uwp/api/windows.globalization.applicationlanguages.primarylanguageoverride)
+property from `Windows.Globalization.ApplicationLanguages` namespace is not supported for
+WindowsAppSDK un-packaged apps, only for packaged. To support the same behavior for un-packaged
+applications `PrimaryLanguageOverride` property is introduced in
+`Microsoft.Windows.ApplicationModel.Resources` MRTCore namespace. This property
+(`Microsoft.Windows.ApplicationModel.Resources.PrimaryLanguageOverride`) overrides the behavior of
+the existing one (`Windows.Globalization.ApplicationLanguages.PrimaryLanguageOverride`).
+
+# API Pages
+
+## ApplicationLanguages class
+
+Specifies the language-related preferences that the app can use and maintain.
+
+# Remarks
+
+The languages referenced in this class are represented by
+[BCP-47](https://tools.ietf.org/html/bcp47) language tags. Each of these is a language identifier
+and not a locale name, although the two structurally can be the same. As a language tag, "en-US"
+means American English (or the dialect of English spoken in the United States). It makes no claims
+about the actual location of the speakers (though many of them do reside in the US) and no other
+claims about conventions that are used in the US such as the measurement system or currency.
+
+Example:
+
+```c#
+Microsoft.Windows.ApplicationModel.Resources.ApplicationLanguages.PrimaryLanguageOverride = "en-US";
+```
+
+Language tags support the Unicode extensions "ca-" and "nu-". (See Unicode Key/Type Definitions.)
+Note that these extensions can affect the numeral system or calendar used by globalization objects.
+
+## ApplicationLanguages.PrimaryLanguageOverride property
+
+Gets or sets an override for the app's preferred language, expressed as a
+[BCP-47 language tag](https://tools.ietf.org/html/bcp47). This setting is global for the running
+process and is not persisted.
```

Comment:
I would call out explicitly here that this is a difference in behavior from the `Windows.Globalization` api.

Reply:
done.Thank you

Reply:
fixed